listenbrainz/listenstore/redis_listenstore.py

```diff
@@ -10,9 +10,8 @@
 
 class RedisListenStore(ListenStore):
 
-    RECENT_LISTENS_KEY = "lb_recent_listens"
-    RECENT_LISTENS_MAX = 100
-    RECENT_LISTENS_MAX_TIME_DIFFERENCE = 300
+    RECENT_LISTENS_KEY = "lb_recent_sorted"
```

Comment:
I changed the name of the key, to make deployment easier.

Reply:
Thanks! :)